plugins/chat/db/migrate/20231227160001_add_type_and_target_id_to_chat_mentions.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
```

Comment:
I would suggest you consider all the index operations in another migration and follow this kind of pattern instead:

```ruby
# frozen_string_literal: true

class AddChatMentionsIndex < ActiveRecord::Migration[7.0]
  disable_ddl_transaction!

  def up
    remove_index :chat_mentions, name: :chat_mentions_index, algorithm: :concurrently
    add_index :chat_mentions,
              [:chat_message_id],
              algorithm: :concurrently
  end

  def down
    add_index :chat_mentions,
              [:chat_message_id, :user_id, :notification_id],
              unique: true,
              name: 'chat_mentions_index',
              algorithm: :concurrently
    remove_index :chat_mentions, column: [:chat_message_id], algorithm: :concurrently
  end
end
```

Reply:
This makes a lot of sense 👍.